content/en/docs/setup/independent/control-plane-flags.md

```diff
@@ -0,0 +1,59 @@
+---
+reviewers:
+- fabriziopandini
+- luxas
+- timothysc 
+title: Customizing control plane flags with kubeadm
```

Comment:
`Customizing control plane configuration with kubeadm` is more generic and future-proof